tests/test_playbooks/external_usergroup.yml

```diff
@@ -0,0 +1,75 @@
+---
+- hosts: localhost
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+  - include_tasks: tasks/location.yml
+    vars:
+      location_state: "present"
+  - include_tasks: tasks/organization.yml
+    vars:
+      organization_state: "present"
+  - include: tasks/usergroup.yml
+    vars:
+      usergroup_name: "internal_group"
+      usergroup_state: present
+  - include: tasks/auth_source_ldap.yml
+    vars:
+      auth_source_ldap_state: present
+      auth_source_ldap_attr_login: dc=vagrant,dc=vm
+      auth_source_ldap_groups_base: cn=groups,cn=accounts,dc=vagrant,dc=vm
+      auth_source_ldap_tls: False
+
+- hosts: tests
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - name: 'Test external usergroup with minimal params (name)'
```

Comment:
I think you can remove this `block` and have the includes directly, as we only have one test block anyways